src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2207,14 +2208,12 @@ public T block(Duration timeout) {
 	 * @param <E> the error type
 	 *
 	 * @return a new {@link Mono}
+	 * @deprecated Use {@link #otherwise(Class, Function)} instead.
```

Comment:
should be `#switchOnError` not `#otherwise`. Maybe also indicate it will be removed in 3.1.0.RELEASE (or any pre-release after M1)?